client/src/components/Nav/index.jsx

```diff
@@ -33,10 +34,25 @@ class Nav extends Component {
                  </React.Fragment>)
   }
 
+  renderNavRecover() {
+    return (
+      <React.Fragment>
+        {
+        <li><Link to={"/recover"}>forgot password?</Link>
+        </li>
```

Comment:
Please use pretty indent for embedded tags. [styleCode](https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-wrap-multilines.md)